src/legacy/server/i18n/i18n_mixin.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n, i18nLoader } from '@kbn/i18n';
+import { basename } from 'path';
+import { Server } from 'hapi';
+import type { UsageCollectionSetup } from '../../../plugins/usage_collection/server';
+import { getKibanaTranslatoinPaths } from './get_kibana_translation_paths';
```

Comment:
nit: Typo
```suggestion
import { getKibanaTranslationPaths } from './get_kibana_translation_paths';
```
and in line 35:
```suggestion
const translationPaths = await getKibanaTranslationPaths(config);
```